extensions/sftp/tests/ListThenFetchSFTPTests.cpp

```diff
@@ -121,7 +120,7 @@ class ListThenFetchSFTPTestsFixture {
     plan->setProperty(list_sftp, "Minimum File Size", "0 B");
     plan->setProperty(list_sftp, "Target System Timestamp Precision", "Seconds");
     plan->setProperty(list_sftp, "Remote Path", "nifi_test/");
-    plan->setProperty(list_sftp, "State File", src_dir + "/state");
+    plan->setProperty(list_sftp, "State File", src_dir.string() + "/state");
```

Comment:
why not `(src_dir / "state").string()`?  (at line 142, too)

Reply:
good idea, https://github.com/apache/nifi-minifi-cpp/pull/1424/commits/eb00f1b247b813448334cb1b9a019164e7e07b28#diff-dfaf51b7f8862c8ffdda0f89bd11d14b8fa83ef70ba0d9e5e686acb1afa51e5fR123